msteams-platform/concepts/build-and-test/tool-sdk-overview.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Tools and SDKs
+author: v-sdhakshina
+description: In this article, learn more about the tools and SDKs available for building your Microsoft Teams app.
+ms.localizationpriority: medium
+ms.topic: overview
+ms.author: surbhigupta
+---
+
+# Tools and SDKs
+
+Teams Platform brings you different SDKs, and APIs to build and develop your apps and there're several libraries available to support various aspects of Teams app development.
+
+The following flow diagram explains you the different SDKs, libraries and its relations:
+
+:::image type="content" source="../../assets/images/Tools-and-SDK-revamp/tools-and-sdk.png" alt-text="Screenshot shows you the list of all tools and SDKs that are available for you to build Teams apps."lightbox="../../assets/images/Tools-and-SDK-revamp/tools-and-sdk.png":::
+
+:::image type="icon" source="../../assets/icons/blue-dot.png" border="false"::: Teams developer platform SDKs require core SDKs to build apps. </br>
```

Comment:
Although I used the word "Core" in the diagram to distinguish with additional libs (like UI libs), let's not use the word in the text, since we haven't exactly defined what core components are. So I'd suggest something like, "A library requires another library to build apps. Your project will automatically pull in another library." 

Please rephrase it nicer to avoid confusion! Also:
- I used a singular "A library" but I am not sure if the plural (like you did originally) is clearer
- "Library" vs "SDK" - I am not sure if we should rather keep calling SDK, not library (But some products are named library, which, I believe has a broader meanings. But again, I am not certain about this)

Reply:
Updated as suggested